### PR TITLE
Make the consent journey 1 page

### PR DESCRIPTION
**Pull request by walaura**
## What does this change?
Makes the consents journey just span 1 page with all its options going one after the other

## Screenshots

screen shot 2018-02-07 at 3 15 33 pm

### CODE REVIEW

File: identity/app/views/consentJourney.scala.html [hunk 1]
```diff
@@ -39,51 +39,48 @@ <h3 class="manage-account__switch-title"></h3>
...
+    @(step.banner.map { title =>
+        Html(s"<p class='form__success'>$title</p>")
+    })
+    <fieldset
+        role="dialog"
+        aria-labelledby="consentWizard@{step.name.capitalize}Title"
+    >
+        <legend id="consentWizard@{step.name.capitalize}Title" class="identity-title identity-title--page-title">@step.title</legend>
+        <div class="identity-forms-wrapper">
+            @if(step.help || showAgeStep(isFirst)) {
```
Comment (rahilb) [inline identity/app/views/consentJourney.scala.html]: I don'y really understand this branch, what does `showAgeStep(isFirst)` mean?

File: identity/app/views/consentJourney.scala.html [hunk 2]
```diff
@@ -39,51 +39,48 @@ <h3 class="manage-account__switch-title"></h3>
...
             </div>
+        </div>
 
-        </fieldset>
-    }
+    </fieldset>
 }
 @renderSteps(steps:List[ConsentStep]) = @{
-    steps.zipWithIndex.map{ case (element, index) => renderStep(element, Some(index)) }
+    val displaySteps = steps.filter(_.show)
```
Comment (mario-galic) [inline identity/app/views/consentJourney.scala.html]: Just inline like so

```
steps
  .filter(_.show)
  .zipWithIndex
  .map(case (element, index) => ...)
```

Comment (walaura) [reply identity/app/views/consentJourney.scala.html]: I need to access the size of the filtered `steps`

File: identity/app/views/consentJourney.scala.html (hunk 1 above)
Comment (walaura) [reply identity/app/views/consentJourney.scala.html]: it's the condition to show the age step or not, i'll clarify that!

**Review by mariogalic — APPROVED** [review]

**Pull request merged**